jaiarobotics/jaiabot
# task/move-engineering-button-into-settings-panel

## Summary
Adds some UI cleanup into the upcoming release. This pull request move the engineering panel into the settings panel. It adds spacing around the panels and the toolbars in the top right corner of the interface. Lastly, it moves the mission speed sliders into the mission panel.

## Testing
1. Open each panel (besides details panels) to verify consistent spacing around borders
2. Run an SRP mission and verify SPR data updates correctly in the accordion
3. Clear local storage and open the mission panel. Move the sliders around and watch the local storage values update correctly
4. Tested on the tablet to ensure the desired look and feel transfers as desired
